test/basic.js

```diff
@@ -53,7 +53,7 @@ t.test('spawn execPath', function (t) {
 })
 
 t.test('exec shebang', function (t) {
-  var child = cp.exec(fixture + ' xyz')
+  var child = spawn(fixture, ['xyz'])
```

Comment:
Oh, that's not it.

Definitely something weird happening with signal-exit, it's sending the signal, but the exit happens with status 0, not the signal.